models/iaf_psc_alpha.cpp

```diff
@@ -321,6 +321,10 @@ iaf_psc_alpha::update( Time const& origin, const long from, const long to )
 
   for ( long lag = from; lag < to; ++lag )
   {
+    // get access to the correct input-buffer entry at the very beginning
+    const index input_buffer_idx = kernel().event_delivery_manager.get_modulo( lag );
+    std::array< double, Buffers_::NUM_INPUT_CHANNELS >& new_input = B_.input_buffer_.get_values( input_buffer_idx );
```

Comment:
Could one replace the complicated type with `auto` here for readability?

Reply:
And is `new_input` the best possible name? How about just `input`?

Reply:
Adapted both and also changed the name of the called function to `get_values_all_channels()`.

Reply:
Also note that the function is now `const` as `input` is no longer directly reset in the neuron model.